src/common/components/form/__tests__/validationUtils.tests.tsx

```diff
@@ -1,6 +1,5 @@
 import {
```

Comment:
This file should probably be renamed to `validationUtils.test.tsx` (omit **s** from "tests")